apps/consent/app/error-handler.ts

```diff
@@ -0,0 +1,39 @@
+import axios from "axios"
+
+interface ErrorResponse {
+  error: boolean
+  message: string
+  responsePayload: null
+}
+
+const errorMessages: { [key: string]: string } = {
+  UserCodeAttemptIpRateLimiterExceededError:
+    "Your rate limit exceeded, please try after some time.",
+  UserCodeAttemptIdentifierRateLimiterExceededError:
+    "Your rate limit exceeded, please try after some time.",
+}
+
+/* eslint @typescript-eslint/ban-ts-comment: "off" */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const handleAxiosError = (err: any): ErrorResponse => {
```

Comment:
```suggestion
export const handleAxiosError = (err: unknown): ErrorResponse => {
```

can you have `unknown` instead?